rhv_checks.py

```diff
@@ -338,6 +340,54 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def is_service_active(ssh, name):
+    """helper function for check_services_running"""
+    stdin, stdout, stderr = ssh.exec_command("systemctl status {}".format(name))
+    status = stdout.read()
+    return "Active: active (running)" in status
+
+
+def check_services_running(system):
+    """Check to see if the vdsm, ovirt-ha-agent, ovirt-ha-broker service is up/running"""
+    hosts = system.api.system_service().hosts_service()
+    hosts_agents = dict()
+    hosts_status = dict()
+    for host in hosts.list():
+        host_service = hosts.host_service(host.id)
+        nics = host_service.nics_service().list()
+        ip_addresses = [n.ip.address for n in nics if n.ip.address.startswith('10.')]
+        ssh = paramiko.SSHClient()
+        ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        for ip in ip_addresses:
+            try:
+                ssh.connect(ip, username="root", password=system.api._password)
+                break
+            except:
```

Comment:
**REQUIRED** : Do you know what exception it throws? Let's try to catch specific exception rather than generic? I suspect its `socket.error` connection timed out?

Reply:
@kedark3 thanks, added